docs/wiki/user/gestures.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Gestures
+---
+
+By default, ACRE2 plays a small animation when the player uses their radios. This is known as a gesture as it only impacts certain parts of the character and doesn't prohibit movement.
+
+To differentiate between using two types radios, by default short-range radios will play a chest touching gesture, long-range radios will play an ear touching gesture. Currently, the radio lists are as follows:
+| Short Range | Long Range |
+|--|--|
+| ACRE_PRC343  | ACRE_PRC148  |
+| ACRE_PRC77   | ACRE_PRC152  |
+| ACRE_SEM52SL | ACRE_PRC117F |
+|              | ACRE_SEM70   |
+
+In a future PR we'll be adding a setter API for you to reorganise these lists or add in your own custom radios. 
```

Comment:
I would not speak about this, this is just committing yourself. :P

Reply:
It's fine, I've got a great track record of not finishing things.

Reply:
But you are committing us, not yourself. ;)